engine/apps/slack/scenarios/alertgroup_appearance.py

```diff
@@ -45,148 +39,17 @@ def process_scenario(self, slack_user_identity, slack_team_identity, payload):
-                        None,
-                    )
-                    if default_values is not None:
-                        default_value = default_values.get(integration)
-                        if default_value is not None:
-                            block["element"]["initial_value"] = default_value
-                blocks.append(block)
-            blocks.append({"type": "divider"})
-
-        common_templates_meta_data = {
-            "source_link": {"placeholder": "{{ payload.link_to_upstream_details }}", "hint": "Jinja2 template."},
-            "grouping_id": {"placeholder": "{{ payload.uid }}", "hint": "Jinja2 template"},
-            "resolve_condition": {
-                "placeholder": '{{ 1 if payload.state == "OK" else 0 }}',
-                "hint": "This Jinja2 template should output one of the following values: ok, true, 1 (case insensitive)",
-            },
-            "acknowledge_condition": {
-                "placeholder": '{{ 1 if payload.state == "OK" else 0 }}',
-                "hint": "This Jinja2 template should output one of the following values: ok, true, 1 (case insensitive)",
             },
-        }
-
-        for common_template in common_templates_meta_data.keys():
-            try:
-                attr = getattr(alert_group.channel, f"{common_template}_template")
-            except AttributeError:
-                continue
-
-            block = {
-                "type": "input",
-                "block_id": f"{common_template}_template",
-                "label": {
-                    "type": "plain_text",
-                    "text": f"{common_template.capitalize().replace('_', ' ')}:",
-                },
-                "element": {
-                    "type": "plain_text_input",
-                    "placeholder": {
-                        "type": "plain_text",
-                        "text": common_templates_meta_data[common_template]["placeholder"],
-                    },
-                    "action_id": UpdateAppearanceStep.routing_uid(),
-                    "multiline": True,
-                },
-                "optional": True,
-                "hint": {
-                    "type": "plain_text",
-                    "text": common_templates_meta_data[common_template]["hint"],
-                },
-            }
-            if attr is not None:
-                block["element"]["initial_value"] = attr
-            else:
-                default_values = getattr(
-                    AlertReceiveChannel, f"INTEGRATION_TO_DEFAULT_{common_template.upper()}_TEMPLATE", None
-                )
-                if default_values is not None:
-                    default_value = default_values.get(integration)
-                    if default_value:
-                        block["element"]["initial_value"] = default_value
-            blocks.append(block)
+            {"type": "section", "text": {"type": "mrkdwn", "text": "Once changed Refresh the alert group"}},
```

Comment:
Do we really need refresh alert group? We don't have this for msteams and telegram

Reply:
We don't want to silently remove the button, so keeping this for transition period and then decide